src/applications/mhv/secure-messaging/components/ThreadList/ThreadsList.jsx

```diff
@@ -17,7 +17,7 @@ const ThreadsList = props => {
     threadsPerPage,
   } = props;
 
-  const MAX_PAGE_LIST_LENGTH = 5;
+  const MAX_PAGE_LIST_LENGTH = 7;
```

Comment:
I am wondering if increasing this may cause issues at high level zoom?

Reply:
The numbers does indeed cut off a bit in super zoom... made the change back to 5.👍

Reply:
Received feedback from A11y -- the recommendation is to set the length to 7 so that the middle numbers are displayed in the pagination component and note that zoom capability goes up to 400%.